Client.xcodeproj/project.pbxproj

```diff
@@ -1732,7 +1732,7 @@
 		};
 		F84B22531A0920C600AAB793 /* Embed App Extensions */ = {
 			isa = PBXCopyFilesBuildPhase;
-			buildActionMask = 12;
+			buildActionMask = 8;
```

Comment:
Nit: I did not see a change in this line, if possible I would leave it as it was..

Reply:
I think this and the one below are for that Embed App Extension work-around you mentioned that let me install. I guess we can try w/o to see.